package.json

```diff
@@ -26,6 +26,7 @@
     "@vue/cli-plugin-typescript": "^4.4.0",
     "@vue/cli-service": "^4.4.0",
     "@walletconnect/web3-provider": "1.2.0-alpha.0",
+    "axios": "^0.20.0",
```

Comment:
We don't need to send a request, we don't need `axios`. We just need to change the verify url with the data populated in the query url like `https://app.mycrypto.com/verify-message?mg=123&address=456...`

Reply:
But I thought that particular `msg`, `sig` and `address` data is currently being stored on IPFS and is not loaded in the application (though definitely possible that I missed it)? Therefore we have to make a request for the external resource.